### DISCUSSION TITLE
备选主页的碎碎念

### DISCUSSION THREAD

**Discussion by MoYuan-CN** [MFn233/PCL-Mainpage-Market#1] [提问]
确认打算放置 [gitee:my_com_cn/pcl_homepage](https://gitee.com/my_com_cn/pcl_homepage) 吗
这个主页的维护.... 只能说不知道多早前开的建议（见 [gitee:my_com_cn/pcl_homepage#I8B1F7](https://gitee.com/my_com_cn/pcl_homepage/issues/I8B1F7)）还烂在那里呢...

**Comment by MFn233** [comment]
我觉得可能他和我一样，没想到自己的主页会有那么多人支持，然后也没啥空去管

**Reply by MFn233** [reply]
不管怎么样，质量不低就能收录啊

**Reply by MoYuan-CN** [reply]
行

**Reply by SYBass** [reply]
放心吧会有初审的

**Reply by SqrtTan** [reply]
~突然想起我在MFn那里开的第一个Issue将近一个月没人管~

**Reply by SYBass** [reply]
不会吧（）
